python/src/trezorlib/cli/cardano.py

```diff
@@ -46,7 +46,17 @@ def cli() -> None:
     "-p", "--protocol-magic", type=int, default=cardano.PROTOCOL_MAGICS["mainnet"]
 )
 @click.option("-N", "--network-id", type=int, default=cardano.NETWORK_IDS["mainnet"])
-@click.option("-t", "--testnet", is_flag=True)
+@click.option(
+    "-t",
+    "--testnet",
+    type=ChoiceType(
```

Comment:
One more nit: I'd introduce a `TESTNET_CHOICES` constant in this file and reuse that.